src/main/groovy/life/qbic/registration/types/DefaultResult.groovy

```diff
@@ -0,0 +1,10 @@
+package life.qbic.registration.types
+
+class DefaultResult {
```

Comment:
What's the purpose of this class? Some JD would help to understand what "default" means in this context 🤔

Reply:
not needed any more and deleted, good point